.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: Bot CI/CD
+
+on:
+  pull_request:
+  push:
+    main
```

Comment:
```suggestion
  push:
    branches:
      - main
```

Reply:
Covered in my comment above

Reply:
8cc0b6b2eadf33c3f6c5808c65743c6983eca822